tests/integration/logging/testCounterPod.yaml

```diff
@@ -15,6 +15,6 @@ spec:
       requests:
         memory: 2Mi
       limits:
-        memory: 10Mi
+        memory: 50Mi
     args: [sh, -c,
-            'for i in `seq 50` ; do echo "$i: logTest-$(date)"; sleep 2; done']
+            'let i=1; while true; do echo "$i: logTest-$(date)"; let i++; sleep 2; done']
```

Comment:
An infinite loop does look dangerous here. How is this pod terminated?

Reply:
Isn't it the case that all pods are terminated after the integration tests are finished?
Anyways, I was making it an infinite loop mainly for debugging. So, I have removed now the infinite loop.